sbi/inference/base.py

```diff
@@ -43,6 +45,9 @@ def __init__(
             device: torch.device on which to compute (optional).
             summary_writer: An optional SummaryWriter to control, among others, log
                 file location (default is <current working directory>/logs.)
+            num_workers: number of parallel workers to start
+            worker_batch_size: how many params are processed on each worker. This number is
```

Comment:
"number of parameters that are processed per worker. A worker will received that many parameters to simulate per call."